### PR TITLE
Categories Page UI/UX

### PR DESCRIPTION
**Pull request by kelkel17**
# 📲 What
- UI/UX implementation for Categories Page

# 🤔 Why
- N/A

# 🛠 How
- Pull the latest commit/changes
- Run the simulator inside xcode (automatic preview)

_Note: In an actual simulator the page cannot be visited yet_

# ✅ Acceptance criteria
- N/A

# ⏰ TODO
- N/A

# 🔗 References
- https://trello.com/c/Q402UG7e
- https://xd.adobe.com/view/88019197-b52c-4c03-a717-e10207152dd7-dd52/screen/034a5fe7-ce99-40a5-adcb-a90ed04fbafd/specs/

https://gyazo.com/1a0e3c25102b31c3cd01485daf962b60

### CODE REVIEW

File: Kwartracker/Resources/fil-PH.lproj/Localizable.strings [hunk 1]
```diff
@@ -71,3 +71,7 @@
 "login_and_security_page.label.email" = "Email Address";
 "login_and_security_page.dummy_data.email" = "email_email_email@emailemail.com";
 "login_and_security_page.dummy_data.password" = "***************";
+
+// Categories pages
+"category_page.navigation_button_item.add_icon" = "AddIcon";
```
Comment (1jacobb1) [inline Kwartracker/Resources/fil-PH.lproj/Localizable.strings:76]: No white space or space?? I think it should be `Add Icon`??

File: Kwartracker/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -71,3 +71,8 @@
 "login_and_security_page.label.email" = "Email Address";
 "login_and_security_page.dummy_data.email" = "email_email_email@emailemail.com";
 "login_and_security_page.dummy_data.password" = "***************";
+
+
+// Categories pages
+"category_page.navigation_button_item.add_icon" = "AddIcon";
```
Comment (1jacobb1) [inline Kwartracker/Resources/en.lproj/Localizable.strings:77]: No white space or space?? I think it should be `Add Icon`??

File: Kwartracker/Scenes/Category Page/Content/CategoryBodyView.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+struct CategoryBodyView: View {
+    @State var search = ""
+    
+    private let shadowRadius: CGFloat = 8
+    private let shadowOffset = CGPoint(x: 6, y: 6)
+    private let rectRadius: CGFloat = 17
+    private let searchButtonSize: CGFloat = 20
+    private let bottomMargin: CGFloat = 8
+    private let bodyMargin: CGFloat = 30
```
Comment (1jacobb1) [inline Kwartracker/Scenes/Category Page/Content/CategoryBodyView.swift:18]: Nice! Prevent from using magic words 👏 👏

File: Kwartracker/Scenes/Category Page/Content/CategoryListView.swift [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        ZStack {
+            HStack {
+                RoundedRectangle(cornerRadius: cornerRadius)
+                    .frame(width: rectangleSize, height: rectangleSize)
+                    .foregroundColor(Color(Asset.Colors.teal.color))
+                    .padding(.leading, verticalMargin)
+                Text(title)
+                    .font(.system(size: fontSize))
+                    .onTapGesture {
+                        withAnimation(.spring().speed(2)) {
```
Comment (1jacobb1) [inline Kwartracker/Scenes/Category Page/Content/CategoryListView.swift:37]: Store `2` to a variable, avoid using magic number rule.

**Review by 1jacobb1 — CHANGES_REQUESTED** [review]

File: Kwartracker/Resources/en.lproj/Localizable.strings (hunk 2 above)
Comment (kelkel17) [reply Kwartracker/Resources/en.lproj/Localizable.strings]: Actually its not needed, I'll remove it

File: Kwartracker/Resources/fil-PH.lproj/Localizable.strings (hunk 1 above)
Comment (kelkel17) [reply Kwartracker/Resources/fil-PH.lproj/Localizable.strings]: animationSpeed

**Review by 1jacobb1 — APPROVED** [review]

**Pull request merged**